docker/README.md

```diff
@@ -39,14 +42,15 @@ Initial setup
 
 There are two different localsettings configurations, depending on whether HQ is running inside a docker container or on your local machine.
```

Comment:
Just for the sake of people understanding what's going on here, maybe add something at the end of this line like "If you are planning on doing local development, it is recommended to run HQ on your local machine, and use docker only for supporting services"